db/docs/source/conf.py

```diff
@@ -1,4 +1,4 @@
-from builtins import str
+# from builtins import str
```

Comment:
If this isn't required, remove the line?

Reply:
Thanks. Done